### PR TITLE
lp_solve a new recipe

### PR DESCRIPTION
**Pull request by mictadlo**

### CODE REVIEW

File: recipes/lp_solve/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    number: 0
+
+test:
+    files:
+      - demo.c
+
+about:
+    home: http://lpsolve.sourceforge.net/5.5/
+    summary: lp_solve is a Mixed Integer Linear Programming (MILP) solver
+    license: LGPL
```
Comment (scopatz) [inline recipes/lp_solve/meta.yaml]: Since this is LGPL, you must include the `license_file` entry

File: recipes/lp_solve/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    version: {{ version }}    
+
+source:
+    fn: {{ name|lower }}_{{ version }}_source.tar.gz
+    url: https://sourceforge.net/projects/lpsolve/files/lpsolve/{{ version }}/{{ name|lower }}_{{ version }}_source.tar.gz
+    md5: 3be57261fc41dd8e210f54017220d5f7
+
+build:
+    number: 0
+
```
Comment (scopatz) [inline recipes/lp_solve/meta.yaml]: Please depend on the `toolchain` package, since this is a C library.

**Comment by scopatz** [comment]
Thanks for putting this in!

File: recipes/lp_solve/meta.yaml (hunk 1 above)
Comment (mictadlo) [inline recipes/lp_solve/meta.yaml]: lp_solve does not come with licence file

File: recipes/lp_solve/meta.yaml (hunk 2 above)
Comment (mictadlo) [inline recipes/lp_solve/meta.yaml]: fixed

File: recipes/lp_solve/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    name: {{ name|lower }}
+    version: {{ version }}    
+
+source:
+    fn: {{ name|lower }}_{{ version }}_source.tar.gz
+    url: https://sourceforge.net/projects/lpsolve/files/lpsolve/{{ version }}/{{ name|lower }}_{{ version }}_source.tar.gz
+    md5: 3be57261fc41dd8e210f54017220d5f7
+
+build:
+    number: 0
```
Comment (mictadlo) [inline recipes/lp_solve/meta.yaml]: fixed

File: recipes/lp_solve/meta.yaml (hunk 1 above)
Comment (scopatz) [inline recipes/lp_solve/meta.yaml]: Then you have to put a copy in the recipe. See http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

File: recipes/lp_solve/meta.yaml (hunk 3 above)
Comment (scopatz) [inline recipes/lp_solve/meta.yaml]: Thanks, though note the linter failure.

**Comment by mictadlo** [comment]
@scopatz Any idea why AppVeor failed?

**Comment by scopatz** [comment]
Since you are skipping windows, you need to add `[skip appveyor]` to the commit message to not have appveyor fail. We need to have better docs for this.  This only affects staged recipes. I am not sure if this can be safely merged without that phrase in the commit message. Mayve someone else on @conda-forge/core knows

**Comment by ocefpaf** [comment]
> Since you are skipping windows, you need to add `[skip appveyor]` to the commit message to not have appveyor fail. We need to have better docs for this. 

`skip  appveyor` saves CI time but won't skip the build in future PRs. You need the `skip: True  # [win]`, which you already have.

> This only affects staged recipes.

Not really. If you don't want `appveyor` to start in a feedstock it will work too and is recommended for testing long build that Windows is not desirable.

> I am not sure if this can be safely merged without that phrase in the commit message. Mayve someone else on @conda-forge/core knows

We can merge this. That failure is due to another commit cancelling the builds. The odd par tis that lastr build is https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.10393 and not https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.10391 that I get when clicking above.

**Comment by scopatz** [comment]
Ok, let's merge then,

**Comment by mictadlo** [comment]
Thank you.

**Pull request merged**